.travis.yml

```diff
@@ -61,7 +61,7 @@ install:
   # Yii2
   - composer create-project "yiisoft/yii2-app-basic:@dev" frameworks-yii-basic --no-dev
   # Phalcon
-  - '[[ "$TRAVIS_PHP_VERSION" == "hhvm" ]] || [[ "$TRAVIS_PHP_VERSION" == "7.0" ]] || git clone -q --depth=1 https://github.com/phalcon/cphalcon.git'
+  - '[[ "$TRAVIS_PHP_VERSION" == "hhvm" ]] || [[ "$TRAVIS_PHP_VERSION" == "7.0" ]] || git clone -b 2.1.x --single-branch -q --depth=1 https://github.com/phalcon/cphalcon.git'
```

Comment:
https://blog.phalconphp.com/post/phalcon-3-0-0-released

I'll try to fix today